conf/configuration.yaml

```diff
@@ -66,13 +66,13 @@ logging:
 ############ DATASOURCE                  ###############
 ########################################################
 # Configuration of the datasource
-elastic-nodes: ${ARLAS_ELASTIC_NODES:-localhost:9300}
-elastic-sniffing: ${ARLAS_ELASTIC_SNIFFING:-false}
-elastic-cluster: ${ARLAS_ELASTIC_CLUSTER:-elasticsearch}
 arlas-index:  ${ARLAS_ELASTIC_INDEX:-.arlas}
-elastic-enable-ssl: ${ARLAS_ELASTIC_ENABLE_SSL:-false}
-elastic-credentials: ${ARLAS_ELASTIC_CREDENTIALS:-user:password}
-elastic-compress: ${ARLAS_ELASTIC_COMPRESS:-true}
+elastic:
```

Comment:
Changing ARLAS-server configuration may be considered as a breaking change, right? Perhaps, we can change configuration keys in a further PR with some other breaking changes (API for example).

cc @MohamedHamouGisaia

Reply:
Indeed it will be a breaking change. Especially for the ones that configure ARLAS-server using the yaml file. Howevern, deployments using ENV Variables will not be affected

I agree, maybe we can change the keys in v13.7.0 of ARLAS-server. What do you think @alainbodiguel

Reply:
In the code, I support both the old and new format. Is that be ok ? Else, there is no problem with postponing this change.

Reply:
My bad, I didn't see that
In this case there is no need to postpone the change :)

Reply:
👍

Reply:
Maybe we could add a note in the documentation to explain that old config keys are deprecated.